src/TamrielOMeter.lua

```diff
@@ -108,13 +108,24 @@ function TamrielOMeter:GetCurrentMapMeasurement()
     local measurement = self:GetMeasurement(mapId)
 
     if (not measurement) then
-        -- try to calculate the measurement if they are not yet available
+        -- try to calculate the measurement if it does not yet exist
         self:CalculateMapMeasurement()
     end
 
     return self.measurements[mapId]
 end
 
+function TamrielOMeter:GetMapMeasurementByMapID(mapId)
```

Comment:
Please change the function name to `GetMapMeasurementByMapId` with a lower-case 'd', so it matches all the other functions in the lib that have 'MapId' in their name.

Reply:
done